pkgs/development/compilers/rust/binary.nix

```diff
@@ -53,16 +53,21 @@ rec {
       patchShebangs .
     '';
 
-    installPhase = ''
-      ./install.sh --prefix=$out \
-        --components=${installComponents}
-
-      # Do NOT, I repeat, DO NOT use `wrapProgram` on $out/bin/rustc
-      # (or similar) here. It causes strange effects where rustc loads
-      # the wrong libraries in a bootstrap-build causing failures that
-      # are very hard to track down. For details, see
-      # https://github.com/rust-lang/rust/issues/34722#issuecomment-232164943
-    '';
+    installPhase =
+      ''
+        ./install.sh --prefix=$out \
+          --components=${installComponents}
+
+        # Do NOT, I repeat, DO NOT use `wrapProgram` on $out/bin/rustc
+        # (or similar) here. It causes strange effects where rustc loads
+        # the wrong libraries in a bootstrap-build causing failures that
+        # are very hard to track down. For details, see
+        # https://github.com/rust-lang/rust/issues/34722#issuecomment-232164943
+      ''
+      + lib.optionalString stdenv.hostPlatform.isDarwin ''
+        install_name_tool -change "/usr/lib/libcurl.4.dylib" \
+        "${curl.out}/lib/libcurl.4.dylib" "$out/bin/cargo"
```

Comment:
lib.getLib?

Reply:
I replaced this and the other instance of `curl.out`. Thanks!